config/compareWarpAssembleCoadd.py

```diff
@@ -1,7 +1,8 @@
 config.badMaskPlanes = ("BAD", "EDGE", "SAT", "INTRP", "NO_DATA",)
-config.subregionSize = (10000, 200) # 200 rows (since patch width is typically < 10k pixels
+config.subregionSize = (10000, 200)  # 200 rows (since patch width is typically < 10k pixels
 config.doMaskBrightObjects = True
+config.removeMaskPlanes.append("CROSSTALK")
 config.doNImage = True
-                                                                                                                                                                        
-config.assembleStaticSkyModel.badMaskPlanes = ("BAD", "EDGE", "SAT", "INTRP", "NO_DATA",)
```

Comment:
Why is `"NO_DATA"` not kept here? The commit message does seem to imply it is.

Reply:
See change in pipe_tasks. The Non-camera-specific defaults is reset there.